spec/controllers/admin/blog/posts_controller_spec.rb

```diff
@@ -95,7 +95,7 @@
 
         post :create, blog_post: invalid_attributes
 
-        expect(page.find("h1")).to have_content "New Blog/Post"
+        expect(page.find("h1")).to have_content "New Blog Post"
```

Comment:
How come this changes?

Reply:
Nested resources were coming off wrong in the search for dashboards, IIRC. This was a change to make it work but I can work around that, if you prefer to keep this behaviour. See `ApplicationHelper#default_resource_name`.

This is actually another question I have: is `Blog/Post` for resource name intentional? Wouldn't this be `Blog Post` instead?